ci/cloudbuild/unit-test.yaml

```diff
@@ -0,0 +1,97 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+# # Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This can be ran by doing the following:
+# gcloud builds submit . \
+# --config=ci/cloudbuild/test.yaml \
+# --ignore-file=ci/cloudbuild/test.ignore \
+# --substitutions='_CLOUDSDK_COMPUTE_ZONE=<ZONE>,_CLOUDSDK_CONTAINER_CLUSTER=<CLUSTER-NAME>'
+#
+# NOTE: Be sure to replace <ZONE> and <CLUSTER-NAME> accordingly.
+#
+# If you want the test to create and delete its own cluster, do the following:
+# gcloud builds submit . \
+# --config=ci/cloudbuild/test.yaml \
+# --ignore-file=ci/cloudbuild/test.ignore \
+# --substitutions='_FULL_RELEASE_BUCKET=<RELEASE-BUCKET>,_DELETE_CLUSTER=true'
+#
+# NOTE: Be sure to replace <RELEASE-BUCKET>. A <RELEASE-BUCKET> is the same
+# used when creating a release. It is the gs:// URI that contains
+# cloud-build/fresh-cluster.yaml and cloud-build/delete-cluster.yaml.
+
+timeout: 7200s
+options:
+  machineType: "E2_HIGHCPU_8"
+substitutions:
+  _GOLANG_IMAGE: "golang:1.17"
+
+steps:
+  - id: verify build
+    waitFor: ["-"]
+    timeout: 900s
+    name: "${_GOLANG_IMAGE}"
+    env:
+      - "GO111MODULE=on"
+      - "GOPROXY=https://proxy.golang.org"
+      - "GOSUMDB=sum.golang.org"
+    entrypoint: bash
+    args:
```

Comment:
I think these can use the "script" field: https://cloud.google.com/build/docs/configuring-builds/run-bash-scripts#using_the_script_field

Reply:
Done.